starboard/build/config/modular/shared/BUILD.gn

```diff
@@ -0,0 +1,224 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//starboard/build/config/os_definitions.gni")
```

Comment:
These should be globally defined, do we have to import them?

Reply:
interesting, removing it doesnt give any error.
I normally treat this as c includes where i include something when the variable inside is being used, ill remove it.